app/src/main/res/layout/topic_revision_summary_view.xml

```diff
@@ -16,9 +16,9 @@
   <com.google.android.material.card.MaterialCardView
     android:layout_width="match_parent"
     android:layout_height="wrap_content"
-    android:layout_marginStart="8dp"
-    android:layout_marginTop="12dp"
-    android:layout_marginEnd="8dp"
+    android:layout_marginStart="@dimen/topic_revision_summary_card_layout_margin_start"
+    android:layout_marginTop="@dimen/topic_revision_summary_card_layout_margin_top"
+    android:layout_marginEnd="@dimen/topic_revision_summary_card_layout_margin_end"
     android:layout_marginBottom="12dp"
     android:clipToPadding="true"
     android:onClick="@{(v) -> viewModel.onRevisionItemPressed.onTopicRevisionSummaryClicked(viewModel.subtopic)}"
```

Comment:
**layout_marginBottom** is actually 0dp for the `layout-sw600dp-port/topic_revision_summary_view.xml` so include this case too. Ditto for the other xmls.